src/components/CategoryFilter/CategoryFilter.Overview.stories.mdx

```diff
@@ -23,14 +23,35 @@ The CategoryFilter component consists of a button with text, typically used in a
 <Canvas isExpanded>
   <Story name="Default">
     {() => {
-      const [isSelected, setSelected] = useState(0);
+      const [isSelected, setIsSelected] = useState({
+        chocolate: false,
+        vanilla: true,
+        strawberry: false,
+      });
+      const handleSelect = selectedKey => {
+        setIsSelected(prevIsSelected => ({
+          ...prevIsSelected,
+          [selectedKey]: !prevIsSelected[selectedKey],
+        }));
+      };
       return (
-        <CategoryFilter
-          isSelected={isSelected}
-          onClick={() => setSelected(prevIsSelected => !prevIsSelected)}
-        >
-          My Category
-        </CategoryFilter>
+        <Box direction="row" childGap="xs">
+          <CategoryFilter
+            isSelected={isSelected.chocolate}
+            onClick={() => handleSelect('chocolate')}
+          >
+            chocolate
+          </CategoryFilter>
+          <CategoryFilter isSelected={isSelected.vanilla} onClick={() => handleSelect('vanilla')}>
+            vanilla
+          </CategoryFilter>
+          <CategoryFilter
+            isSelected={isSelected.strawberry}
+            onClick={() => handleSelect('strawberry')}
+          >
+            strawberry
+          </CategoryFilter>
+        </Box>
```

Comment:
making this demo match it's intended use by adding more than one CategoryFilter button